### ISSUE TITLE
Error Message During Combat as Blood DK

### ISSUE THREAD

**Issue by Devin6519** [Hekili/hekili#436]
**BEFORE SUBMITTING AN ISSUE:**

1.  Check to see if you are using the latest version of the addon.  If there is a newer alpha/beta/release version, see if your problem exists with the new version.

2.  Check to see if someone else has already opened an issue report here.  Please don't submit repeat reports.

**Describe the Bug/Issue**
This is the Error I get every time I enter combat as Blood DK: 

Interface\AddOns\Hekili\Classes\DeathKnightBlood.lua:720: attempt to index global 'active_Dot' (a nil value)
Time: Thu Oct  1 20:04:59 2020
Count: 12885
Stack: Interface\AddOns\Hekili\Classes\DeathKnightBlood.lua:720: attempt to index global 'active_Dot' (a nil value)
[string "=[C]"]: ?
[string "@Interface\AddOns\Hekili\Classes\DeathKnightBlood.lua"]:720: in function `handler'
[string "@Interface\AddOns\Hekili\State.lua"]:5103: in function `RunHandler'
[string "@Interface\AddOns\Hekili\Core.lua"]:1578: in function `ProcessHooks'
[string "@Interface\AddOns\Hekili\UI.lua"]:825: in function <Interface\AddOns\Hekili\UI.lua:689>

Locals: 
**To Reproduce**
- Go to '...'
- Click on '....'
- Scroll down to '....'
- See error

**Expected behavior**

**Screenshots**

**WHEN SUBMITTING AN ISSUE:**

Please provide the following information.  Leaving it out may result in your ticket being ignored (because I won't have the information needed to explore your report).

1.  The information from the Issue Reporting tab ( /hekili > Issue Reporting ).  This provides your talent and gear information.  You can paste it to pastebin.com and provide a link here.

**PUT ISSUE REPORT LINK HERE:**

2.  If reporting a disagreement with the addon's recommendations, please take a snapshot when you observe the issue and provide the snapshot.  This is not a screenshot.  A snapshot can be generated by binding the Pause feature on the Toggles tab.  When you press Pause, a snapshot of how the addon decided on its CURRENT recommendations is generated and saved to the Snapshots tab.  Copy and paste this to pastebin.com and provide the link here.
 
**PUT SNAPSHOT LINK HERE:**

3.  If reporting an error (i.e., the addon freezes and/or disables itself), please install the BugSack and BugGrabber addons as they will enable you to paste the exact error message, with detail that will help me identify where the code error is located.  You can paste the error information here or use pastebin.com for that as well.

**PUT ERROR MESSAGE(S) HERE:**

**Comment by Hekili** [comment]
Thanks, this is due to a typo that I'll resolve in the next build.

Just a quick reminder, the rest of the instructions are still important when reporting a bug.  I need to be able to see which version of the addon you're using, for instance.  But I found the error already.  Thanks!

**Comment by Devin6519** [comment]
Sorry about that I’ll make sure to include it next time if I need to report something. Sent from Mail for Windows 10 From: HekiliSent: Thursday, October 1, 2020 8:16 PMTo: Hekili/hekiliCc: Devin6519; AuthorSubject: Re: [Hekili/hekili] Error Message During Combat as Blood DK (#436) Thanks, this is due to a typo that I'll resolve in the next build.Just a quick reminder, the rest of the instructions are still important when reporting a bug. I need to be able to see which version of the addon you're using, for instance. But I found the error already. Thanks!—You are receiving this because you authored the thread.Reply to this email directly, view it on GitHub, or unsubscribe.

**Comment by Devin6519** [comment]
Oh. I failed to mention in the lasted build(49)it suggest Death and Decay followed by Death’s Due when of the Night Fae Covenant Sent from Mail for Windows 10 From: HekiliSent: Thursday, October 1, 2020 8:16 PMTo: Hekili/hekiliCc: Devin6519; AuthorSubject: Re: [Hekili/hekili] Error Message During Combat as Blood DK (#436) Thanks, this is due to a typo that I'll resolve in the next build.Just a quick reminder, the rest of the instructions are still important when reporting a bug. I need to be able to see which version of the addon you're using, for instance. But I found the error already. Thanks!—You are receiving this because you authored the thread.Reply to this email directly, view it on GitHub, or unsubscribe.
